app/src/components/icons.tsx

```diff
@@ -338,3 +338,399 @@ export const DataConnectIcon = () => (
+          fill="#7A7B9A"
+        />
+      </g>
+    </g>
+  </svg>
+);
+
+export const IndustrialIcon = () => (
+  <svg
+    width="24"
+    height="24"
+    viewBox="0 0 24 24"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <g id="Icons">
+      <mask
+        id="mask0_4581_38062"
+        style={{ maskType: "alpha" }}
+        maskUnits="userSpaceOnUse"
+        x="0"
+        y="0"
+        width="24"
+        height="24"
+      >
+        <rect id="Bounding box" width="24" height="24" fill="#D9D9D9" />
+      </mask>
+      <g mask="url(#mask0_4581_38062)">
+        <path
+          id="apparel"
+          d="M5.99896 10.95L4.99896 11.5C4.76562 11.6333 4.51562 11.6667 4.24896 11.6C3.98229 11.5333 3.78229 11.3833 3.64896 11.15L1.64896 7.65C1.51562 7.41667 1.48229 7.16667 1.54896 6.9C1.61563 6.63333 1.76562 6.43333 1.99896 6.3L7.74896 3H9.49896C9.64896 3 9.76979 3.04583 9.86146 3.1375C9.95312 3.22917 9.99896 3.35 9.99896 3.5V4C9.99896 4.55 10.1948 5.02083 10.5865 5.4125C10.9781 5.80417 11.449 6 11.999 6C12.549 6 13.0198 5.80417 13.4115 5.4125C13.8031 5.02083 13.999 4.55 13.999 4V3.5C13.999 3.35 14.0448 3.22917 14.1365 3.1375C14.2281 3.04583 14.349 3 14.499 3H16.249L21.999 6.3C22.2323 6.43333 22.3823 6.63333 22.449 6.9C22.5156 7.16667 22.4823 7.41667 22.349 7.65L20.349 11.15C20.2156 11.3833 20.0198 11.5292 19.7615 11.5875C19.5031 11.6458 19.249 11.6083 18.999 11.475L17.999 10.975V20C17.999 20.2833 17.9031 20.5208 17.7115 20.7125C17.5198 20.9042 17.2823 21 16.999 21H6.99896C6.71562 21 6.47813 20.9042 6.28646 20.7125C6.09479 20.5208 5.99896 20.2833 5.99896 20V10.95ZM7.99896 7.6V19H15.999V7.6L19.099 9.3L20.149 7.55L15.849 5.05C15.599 5.9 15.1281 6.60417 14.4365 7.1625C13.7448 7.72083 12.9323 8 11.999 8C11.0656 8 10.2531 7.72083 9.56146 7.1625C8.86979 6.60417 8.39896 5.9 8.14896 5.05L3.84896 7.55L4.89896 9.3L7.99896 7.6Z"
+          fill="#7A7B9A"
+        />
+      </g>
+    </g>
+  </svg>
+);
```

Comment:
Just a comment for the future. We could looking into having a better process of dealing with our icons.
- Our Icons should have uniform sizing
- We should generate an icon sprite
- Geneate a single icon component `<Icon name=[icon_name] />`

Reply:
Creating a ticket for this :+1: